packages/fast-components-react-base/src/viewport-positioner/README.md

```diff
@@ -8,6 +8,8 @@ The *viewport positioner* and the anchor element it is attached to must both be
 
 If no "anchor" is specified the component will be disabled and not attempt to adjust layout. If no viewport is specified the component will assume the viewport is the root document element of the page.
 
+Authors should ensure that the positioner element has fixed dimensions, most likely coming from css styles, sizing forced by a parent element (i.e. grid cell), etc....  
```

Comment:
This is in regards to having a fixed width/height? Can you provide some insight to me as to why that’s important? Seems arbitrary without context.

Also, three dots for ellipsis :)

Reply:
I worry that there could be situations with a resizing positioner where it gets placed, the new placement causes the dimensions to change, which causes an intersection and recalculation, which places it back in the first position, which causes an intersection and restarts the cycle.  I don't want to say outright that people should not do it, just that it makes it more likely to have problems.

Reply:
I'll use what Jane said.